index.html

```diff
@@ -9,5 +9,16 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
```

Comment:
I would add to the body a style like `body-container`in order to establish a `padding` separating the beginning of the page from the content.